### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by TekExplorer** [Rohan-Bansal/dart-canvas-api#1]
Hello! just wanted to see what the status of this is, as I would like to integrate a discord bot to track assignments and issue reminders and such programmatically, however, to do that I need an API client to communicate with.

**Comment by rohan-bansal** [comment]
Unfortunately, this project has been put on hold with no expected updates for the foreseeable future. I'm a student right now and have moved on to other interests. However, any PRs are welcome!

Note: This API is meant for Flutter integration (I don't see how a discord bot would use this as a client), but if you mean to create a Dart server environment then ignore this.

**Comment by TekExplorer** [comment]
It's just an API client, isn't it? So it could support Dart apps - such as a Discord bot made in Dart

**Comment by rohan-bansal** [comment]
True, but using Dart as a language for a bot wouldn't be optimal.
